sonar-plugin-api/src/main/java/org/sonar/api/platform/Server.java

```diff
@@ -47,6 +45,12 @@
 
   public abstract String getContextPath();
 
+  public abstract String getPublicRootUrl();
+
+  public abstract boolean isDev();
+
+  public abstract boolean isSecured();
```

Comment:
Missing @since 5.4 and Javadoc (is it when `getPublicRootUrl` starts with `https` ?).